src/checkers/inference/InferenceAnnotatedTypeFactory.java

```diff
@@ -70,6 +70,8 @@
 import com.sun.source.tree.MethodInvocationTree;
 import com.sun.source.tree.NewClassTree;
 import com.sun.source.tree.Tree;
+import org.checkerframework.javacutil.TypesUtils;
```

Comment:
These two imports seem unused. Can they be removed?
If they are needed, can you add them to the `org.checkerframework...` section earlier in the file?

Reply:
Sure, removed. Sometimes we should add spotless to prevent un used import automatically.

Reply:
Yes, once we're done with the updates, we should get back to #430.